README.rst

```diff
@@ -23,11 +37,11 @@ Tasks
 _____
 
 Most of these methods will return a `scaleapi.Task` object, which will contain information
-about the json response (task_id, status, etc.).
+about the json response (task_id, status, params, response etc.).
```

Comment:
Smallest nitpick in the world, add a comma between response and etc. `response, etc.`

Reply:
Should've run grammarly on readme as well :) Thanks for the catch!